packages/firestore/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# v4.6.0
+- [fixed] Fixes validation of nested arrays to allow indirect nesting
+
```

Comment:
There was a change in 4.5.1 too.  You can copy/paste from https://firebase.google.com/support/release-notes/js#4.5.1

Reply:
Good catch! Done.